src/bundle/Resources/public/js/scripts/fieldType/ezobjectrelationlist.js

```diff
@@ -77,9 +77,34 @@
                 const { escapeHTML } = ibexa.helpers.text;
                 const itemNodes = relationsContainer.querySelectorAll('.ibexa-relations__item');
                 const itemNode = itemNodes[itemNodes.length - 1];
-
-                itemNode.setAttribute('data-content-id', escapeHTML(item.ContentInfo.Content._id));
+                const contentId = escapeHTML(item.ContentInfo.Content._id);
+                const locationId = item.id;
+                const currentVersionNo = item.ContentInfo.Content.CurrentVersion.Version.VersionInfo.versionNo;
+                const languageCodes = item.ContentInfo.Content.CurrentVersion.Version.VersionInfo.VersionTranslationInfo.Language.map(
+                    (language) => language.languageCode,
+                );
```

Comment:
what would you say on putting `item.ContentInfo.Content.CurrentVersion.Version.VersionInfo` into extra, much shorter variable? 
```suggestion
                const { VersionInfo } = item.ContentInfo.Content.CurrentVersion.Version;
                const currentVersionNo = VersionInfo.versionNo;
                const languageCodes = VersionInfo.VersionTranslationInfo.Language.map(
                    (language) => language.languageCode,
                );
```